### PR TITLE
Add ARM64 nodesets

### PR DESCRIPTION
**Pull request by mrhillsman**
VEXXHOST only

1. ubuntu-xenial-vxh-arm64
2. ubuntu-bionic-vxh-arm64
3. centos-7-vxh-arm64

Fixes theopenlab/openlab#198
Related-Bug: theopenlab/openlab#194

### CODE REVIEW

**Comment by liu-sheng** [comment]
Hi Melvin, as we have reached an agreement, we should open all these issues in `https://github.com/theopenlab/openlab/issues`

**Comment by kiwik** [comment]
Like @liu-sheng say, please ensure all of these nodesets have been availiable in nodepool, then we can merge the PR.
And I think we should remove `vxh` in nodeset name , s/centos-7-vxh-arm64/centos-7-arm64/ , might other cloud vendor provide ARM resource for the label in the future, if it's not necessary, we should not associate with specified vendor.

**Comment by kiwik** [comment]
@mrhillsman please go on your PR, thanks https://github.com/theopenlab/openlab/issues/198#issuecomment-473180894

File: zuul.d/nodesets.yaml [hunk 1]
```diff
@@ -24,12 +24,24 @@
       - name: ubuntu-xenial-vexxhost
         label: ubuntu-xenial-vexxhost
 
+- nodeset:
+    name: ubuntu-xenial-vxh-arm64
```
Comment (kiwik) [inline zuul.d/nodesets.yaml]: Only `ubuntu-xenial-arm64` is available now in OpenLab, please update nodeset name and remove the following two ARM nodeset, thx.

**Comment by Yikun** [comment]
Do we still these nodesets.yaml? I prefer we could add this nodesets setting when we really need it. Otherwise, we should maintenance many redundant settings.

**Comment by kiwik** [comment]
> Do we still need these in nodesets.yaml? I prefer we could add this nodesets setting when we really need it. Otherwise, we should maintenance many redundant settings.

We hope to keep consist between Nodepool and Zuul, so reasonable steps is：
1. Config, test and enable labels in Nodepool side
2. Add available labels in openlab-zuul-jobs side

**Pull request closed without merging**